packages/@react-spectrum/labeledvalue/src/LabeledValue.tsx

```diff
@@ -92,7 +102,11 @@ function LabeledValue<T extends SpectrumLabeledValueTypes>(props: SpectrumLabele
       <div
         className={classNames(labelStyles, 'spectrum-Field-wrapper')}>
         <div
-          className={classNames(labelStyles, 'spectrum-Field-field')}>
+          className={classNames(labelStyles, 'spectrum-Field-field')}
+          aria-label={props['aria-label']}
```

Comment:
not sure we need these? they don't do anything on a div with no role is my understanding

I'd attempted to use some of these earlier with VO not reading off anything which is why they weren't being passed through.

Reply:
They do actually! VoiceOver announces it as a group containing the text.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label):

> The aria-label attribute can be used with regular, semantic HTML elements; it is not limited to elements that have an [ARIA role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) assigned.

That said, I'm not sure if anyone will use this. I just saw it in the types and it didn't work.

Reply:
🤯 i swear i tried this and got absolutely nothing, I'll try again

maybe it was only aria-describedby/aria-labelledby

Reply:
Is there a different shortcut? I just tried the "no visible label" in safari with VO and after clicking on the text, it just read Hello, it did not read the label.
Same thing with describedby and labelledby

Reply:
You have to navigate into it with the screen reader cursor, e.g. ctrl + option + left/right, not click directly on it. You'll first reach a group with the label, and then the text itself.